test/release/examples/primers/sparse.good

```diff
@@ -70,50 +70,50 @@ Positions that are members in the sparse domain are marked by a '*':
 * . . . . . . . * 
 
 Iterating over spsDom and indexing into spsArr:
-spsArr((1, 1)) = 1.1
-spsArr((1, 9)) = 1.9
-spsArr((9, 1)) = 9.1
-spsArr((9, 9)) = 9.9
+spsArr((0, 0)) = 0.0
+spsArr((0, 8)) = 0.8
+spsArr((8, 0)) = 8.0
+spsArr((8, 8)) = 8.8
 
 Iterating over spsArr:
-1.1 
-1.9 
-9.1 
-9.9 
+0.0 
+0.8 
+8.0 
+8.8 
 
-the sum of the sparse elements is: 22.0
-the sum of the dense elements is: 614.9
+the sum of the sparse elements is: 17.6
+the sum of the dense elements is: 610.5
 
 Printing spsArr after resetting and adding the diagonal:
-1.1 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 
-0.0 2.2 0.0 0.0 0.0 0.0 0.0 0.0 0.0 
-0.0 0.0 3.3 0.0 0.0 0.0 0.0 0.0 0.0 
-0.0 0.0 0.0 4.4 0.0 0.0 0.0 0.0 0.0 
-0.0 0.0 0.0 0.0 5.5 0.0 0.0 0.0 0.0 
-0.0 0.0 0.0 0.0 0.0 6.6 0.0 0.0 0.0 
-0.0 0.0 0.0 0.0 0.0 0.0 7.7 0.0 0.0 
-0.0 0.0 0.0 0.0 0.0 0.0 0.0 8.8 0.0 
-0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 9.9 
+0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 
+0.0 1.1 0.0 0.0 0.0 0.0 0.0 0.0 0.0 
+0.0 0.0 2.2 0.0 0.0 0.0 0.0 0.0 0.0 
+0.0 0.0 0.0 3.3 0.0 0.0 0.0 0.0 0.0 
+0.0 0.0 0.0 0.0 4.4 0.0 0.0 0.0 0.0 
+0.0 0.0 0.0 0.0 0.0 5.5 0.0 0.0 0.0 
+0.0 0.0 0.0 0.0 0.0 0.0 6.6 0.0 0.0 
+0.0 0.0 0.0 0.0 0.0 0.0 0.0 7.7 0.0 
+0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 8.8 
```

Comment:
(keeping the bulk of the primer 1-based also has the nice benefit of differentiating values that are `0.0` in the output because they're "zero"/IRV values from those that were set to `0.0` because they're in row/column 0, as in this diff).

Reply:
Yep.  We have now reverted back to the original sparse primer.